### ISSUE TITLE
Discuss UI

### ISSUE THREAD

**Issue by cburgdorf** [ninya-io/ninya.io#21]
@timhartmann came up with this mockup. Would like to discuss this a bit. 

![image](https://f.cloud.github.com/assets/521109/2181060/2c51d030-9741-11e3-8c79-559d115c87af.png)

Quoting @herschel666 from twitter.

> I think the vertical divider is dispensable. And I would change the `<h4>`s to clickable, vertically aligned `<label>`-elements.

https://twitter.com/Herschel_R/status/434984871832735744

Emanuel can you elaborate a bit on that. We changed the UI because nobody groked the tag based input (e.g. _location: hannover, hanover answers: javascript, html_). So Tim came up with a mockup of a redesigned dual input which I quite liked. Any other ideas for that?

**Comment by 0x-r4bbit** [comment]
Hey that looks great! I'm sure this would perfectly work with "card" modules when getting search results :) @timhartmann how about adding some mocks for that too?

**Comment by herschel666** [comment]
Don't mind my opinion on the vertical divider. But I think the input-labels should be actual `<label>`-elements with appropriate `for`-attributes, to get a better UX. And I think, that the inputs and the labels fit in one row, if the screen i wide enough. Though this is also my subjective valuation.

Apart from that I like the mockup and I totally agree with you, that the input-section needs a re-work to not confuse the users regarding the dual tag-structure.

**Comment by cburgdorf** [comment]
@herschel666 you are totally right about making them `<label>` with `for` attributes. That was sloppy ;-)

**Comment by timhartmann** [comment]
@PascalPrecht I can do it - the design, which Christoph added here, was developed with the Dev-Tools in Chrome. No photoshop :godmode:

**Comment by 0x-r4bbit** [comment]
Now let's make a great card module for stackwho, something in between G+ cards and Twitter Cards would be great I think.

**Comment by timhartmann** [comment]
@PascalPrecht  :+1:

**Comment by timhartmann** [comment]
Guys, I designed something in photoshop. What do you think about this proposal? 

![index_140218_1](https://f.cloud.github.com/assets/986401/2193332/0a80db96-987b-11e3-9a1f-56a496d629bf.JPG)

**Comment by 0x-r4bbit** [comment]
Maybe you can make the avatar round? (fancy times you know).

Also, we currently only rely on total reputation, to get a more precise result we have to evaluate reputation and upvotes per answer. Maybe you can somehow put the information the card.

So what we basically need is a number for reputation for each tag (?)

All in all, I like the card!

**Comment by 0x-r4bbit** [comment]
But the header is like shit, so i'm just ignoring it. We could also think about extending the card module with additional functionality. For example flipping it, when clicking on it and showing more information about the user. But that's just future stuff.

**Comment by cburgdorf** [comment]
I think this whole thing should be a bit wider (like it was in the first mock). Especially if you want to enter longer search strings like _San Francisco, Berkeley, Oakland_ + _angularjs, nodejs_ it helps if you have a wider box.

Regarding the twitter share button. Currently the way it works is that you can share _the current result_ which is awesome. I think we should communicate that stronger. By moving them into the upper right corner it feels more like "Share this site".

**Comment by 0x-r4bbit** [comment]
Just talked to @cburgdorf about it and another option would be to introduce a two column layout for the cards.

**Comment by timhartmann** [comment]
@PascalPrecht If you like it's shit then don't bleat, but give some better feedback to improve it...

**Comment by 0x-r4bbit** [comment]
No i'm just ignoring it for now since we're talking about the card module first. Forget the header for now.

**Comment by timhartmann** [comment]
Here is a quick view of the cards - two columns now.
![index_140218_2](https://f.cloud.github.com/assets/986401/2194534/290a80e6-988d-11e3-8b86-26ed0272955b.JPG)

**Comment by 0x-r4bbit** [comment]
@timhartmann Perfect! Now I'd propose to set them not side by side but kinda shifted.

**Comment by timhartmann** [comment]
@PascalPrecht Shifted sounds good. If all are happy with this kind of solution, then let us build it :)

**Comment by 0x-r4bbit** [comment]
How about providing a simple mock first?

**Comment by cburgdorf** [comment]
When we shift them, how about filling the spaces with a huge, low-contrast index number?

Like that:

![untitled drawing](https://f.cloud.github.com/assets/521109/2195069/b4fabed2-9896-11e3-8283-3f42bdd43fae.jpg)

**Comment by cburgdorf** [comment]
@timhartmann another thing noted. In your mocks the cards only use two tags. We should show at least 5 - 10 tags I think. Another reason for making them a little wider. I would like to push the issue :)
